### PR TITLE
[CTOR-1088] Refresh network-firewalls-paloalto-standard-ssh doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md [hunk 1]
```diff
@@ -5,186 +5,313 @@ title: Palo Alto firewall SSH
...
-    --ssh-username=centreon \
-    --ssh-password='centreon-password' \
-    --ssh-backend=sshcli \
-    --component='.*' \
-    --verbose
+	--plugin=network::paloalto::ssh::plugin \
+	--mode=environment \
+	--hostname='10.0.0.1' \
+	--ssh-backend='sshcli' \
+	--ssh-username='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md:307]: username & password

File: pp/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md [hunk 2]
```diff
@@ -5,183 +5,308 @@ title: Palo Alto firewall SSH
...
+<TabItem value="Net-PaloAlto-Standard-SSH-custom" label="Net-PaloAlto-Standard-SSH-custom">
 
-| Rule name                                     | Description                                                              |
-| :-------------------------------------------- | :----------------------------------------------------------------------- |
-| Net-PaloAlto-Standard-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errors and discards              |
-| Net-PaloAlto-Standard-SNMP-Traffic-Name       | Discover network interfaces and monitor status and bandwidth utilization |
+| Service Alias | Service Template                             | Service Description      |
+|:--------------|:---------------------------------------------|:-------------------------|
+| Environment   | Net-PaloAlto-Standard-Environment-SSH-custom | Check components         |
+| Ha            | Net-PaloAlto-Standard-Ha-SSH-custom          | Check high-availability  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md:24]:
```suggestion
| Ha            | Net-PaloAlto-Standard-Ha-SSH-custom          | Check high availability  |
```

File: pp/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md [hunk 3]
```diff
@@ -5,183 +5,308 @@ title: Palo Alto firewall SSH
...
-    --ssh-username=centreon \
-    --ssh-password='centreon-password' \
-    --ssh-backend=sshcli \
-    --component='.*' \
-    --verbose
+	--plugin=network::paloalto::ssh::plugin \
+	--mode=environment \
+	--hostname='10.0.0.1' \
+	--ssh-backend='sshcli' \
+	--ssh-username='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md:302]: username & password

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md [hunk 4]
```diff
@@ -5,186 +5,313 @@ title: Palo Alto firewall SSH
...
+	--ssh-password='' \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md:308]:
```suggestion
	--ssh-username='XXXX' \
	--ssh-password='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md [hunk 5]
```diff
@@ -5,183 +5,308 @@ title: Palo Alto firewall SSH
...
+	--ssh-password='' \
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-firewalls-paloalto-standard-ssh.md:303]:
```suggestion
	--ssh-username='XXXX' \
	--ssh-password='XXXX' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**